schema-derivatives-options.graphql

```diff
@@ -1333,12 +1378,66 @@ type Position @entity {
   " Underlying amount (in USD) at the time the option was taken "
   amountUSD: BigDecimal!
 
-  " Price the asset actually is at maturity "
-  exercisedPriceUSD: BigDecimal
```

Comment:
Do we lose anything by getting rid of this?

Reply:
same w strike price

Reply:
Expiration price and strike price are under Option. In the case of Opyn at least, it's not really possible to update all out-of-money positions after expiration, since there is no event emitted in that case. This way there is just one field to update when the option expires.